docs/dev/setup.sh

```diff
@@ -3,7 +3,7 @@
 OSTYPE=$(uname -s)
 COMPOSE_CMD='docker-compose'
 CLUSTER_NAME="demo-a"
-COMPOSE_FILE="./docs/dev/armada-compose.yaml"
```

Comment:
The previous version is correct.  @richscott changed this file to use his docker-file.  And I reference this file in a different part of the build.  

We could remove this and only use @suprjinx localdev code but let's leave this alone for this PR.

Reply:
Didn't realize that we removed this in a previous PR.  This is actually correct!